NEWS.md

```diff
@@ -1,3 +1,7 @@
+1.30.1 (August 20, 2015)
```

Comment:
Let's correct this to `1.31.0 (unreleased)`, and I'll merge. Thank you very uch!